run-tests.php

```diff
@@ -3282,56 +3217,14 @@ function parse_conflicts(string $text): array
 
 function show_result($result, $tested, $tested_file, $extra = '', $temp_filenames = null)
 {
-    global $html_output, $html_file, $temp_target, $temp_urlbase, $line_length, $SHOW_ONLY_GROUPS;
+    global $temp_target, $temp_urlbase, $line_length, $SHOW_ONLY_GROUPS;
 
     if (!$SHOW_ONLY_GROUPS || in_array($result, $SHOW_ONLY_GROUPS)) {
         echo "$result $tested [$tested_file] $extra\n";
     } elseif (!$SHOW_ONLY_GROUPS) {
         clear_show_test();
     }
 
-    if ($html_output) {
-        if (isset($temp_filenames['file']) && file_exists($temp_filenames['file'])) {
-            $url = str_replace($temp_target, $temp_urlbase, $temp_filenames['file']);
```

Comment:
It looks like this code block was the only place where `$temp_urlbase` was used. As such, we should also remove this global variable, as well as the `--temp-urlbase` option.

Reply:
@nikic noted. I will come back with another PR to remove this PR and ensure there's no regressions. Thanks